bindings/java/c/host.c

```diff
@@ -254,23 +258,28 @@ static size_t copy_code_fn(struct evmc_host_context* context,
 
     // set java method params
     jbyteArray jaddress = CopyDataToJava(jenv, address, sizeof(struct evmc_address));
-    jint jcode_offset = (jint)code_offset;
 
     // call java method
-    jobject jresult = (*jenv)->CallStaticObjectMethod(jenv, host_class, method, context->index,
-                                                      jaddress, jcode_offset);
+    jobject jresult =
+        (*jenv)->CallStaticObjectMethod(jenv, host_class, method, context->index, jaddress);
     assert(jresult != NULL);
 
     // copy jresult back to buffer_data
-    buffer_data = (uint8_t*)(*jenv)->GetDirectBufferAddress(jenv, jresult);
-    (void)buffer_data;
-    assert(buffer_data != NULL);
+    uint8_t* code = (uint8_t*)(*jenv)->GetDirectBufferAddress(jenv, jresult);
```

Comment:
This should use `GetDirectBuffer` from #552.

Reply:
Looks unfinished.

Reply:
You can ignore that, still waiting for an answer on this: https://github.com/ethereum/evmc/pull/545#issuecomment-705864279